signer/src/config/mod.rs

```diff
@@ -747,17 +793,9 @@ mod tests {
                 .rpc_endpoints
                 .contains(&url("http://foo:bar@localhost:5678"))
         );
-        assert!(
-            settings
-                .bitcoin
-                .block_hash_stream_endpoints
-                .contains(&url("tcp://localhost:1234"))
-        );
```

Comment:
nit: there are still some occurrences around of `block_hash_stream_endpoints` (`signer-config.toml.sample`, `signer-*.env` and in here)